Features/AdminCommands/UnMuteCommand.cs

```diff
@@ -0,0 +1,34 @@
+using ArtcordAdminBot.Features.Helpers;
+using DSharpPlus.Commands;
+using DSharpPlus.Entities;
+
+namespace ArtcordAdminBot.Features.AdminCommands
+{
+    public partial class AdminCommandGroup
+    {
+        [Command("unmute")]
+        [System.ComponentModel.Description("Unmutes a user")]
+        public async Task UnMuteAsync(CommandContext ctx,
+        [System.ComponentModel.Description("The user to Unmute.")] DiscordUser targetUser)
+        {
+            var mutedRoleId = await _databaseService.GetMutedRoleAsync(ctx.Guild!.Id);
```

Comment:
again, no `var` for primitive types pls
```suggestion
            ulong mutedRoleId = await _databaseService.GetMutedRoleAsync(ctx.Guild!.Id);
```